v0-991/learn/tools-ides/intellij-plugin.md

```diff
@@ -101,28 +104,28 @@ After obtaining the ZIP file using either of the above approaches, follow the st
 
 ## Using the plugin
 
-For information on using the IntelliJ Ballerina plugin to write Ballerina programs, see [Using the IntelliJ Ballerina plugin](tools-ides/intellij-plugin/using-the-intellij-plugin).
+For information on using the IntelliJ Ballerina plugin to write Ballerina programs, see [Using the IntelliJ Ballerina plugin](/learn/tools-ides/intellij-plugin/using-the-intellij-plugin).
 
 ## Using the features of the plugin
 
 Click on the below links to find information on the various capabilities that are facilitated by the IntelliJ Ballerina plugin for the development process.
 
-- [Running Ballerina programs](tools-ides/intellij-plugin/using-intellij-plugin-features#running-ballerina-programs)
-- [Debugging Ballerina programs](tools-ides/intellij-plugin/using-intellij-plugin-features#debugging-ballerina-programs)
-- [Viewing the sequence diagram](tools-ides/intellij-plugin/using-intellij-plugin-features#viewing-the-sequence-diagram)
-- [Importing modules on the fly](tools-ides/intellij-plugin/using-intellij-plugin-features#importing-modules-on-the-fly)
-- [Importing unambiguous modules](tools-ides/intellij-plugin/using-intellij-plugin-features#importing-unambiguous-modules)
-- [Finding usages](tools-ides/intellij-plugin/using-intellij-plugin-features#finding-usages)
-- [Viewing definitions](tools-ides/intellij-plugin/using-intellij-plugin-features#viewing-definitions)
-- [Formatting Ballerina codes](tools-ides/intellij-plugin/using-intellij-plugin-features#formatting-ballerina-codes)
-- [Viewing details of parameters](tools-ides/intellij-plugin/using-intellij-plugin-features#viewing-details-of-parameters)
-- [Viewing documentation](tools-ides/intellij-plugin/using-intellij-plugin-features#viewing-documentation)
-- [Adding annotation fields via suggestions](tools-ides/intellij-plugin/using-intellij-plugin-features#adding-annotation-fields-via-suggestions)
-- [Using file templates](tools-ides/intellij-plugin/using-intellij-plugin-features#using-file-templates)
-- [Using code snippet templates](tools-ides/intellij-plugin/using-intellij-plugin-features#using-code-snippet-templates)
-- [Checking spellings](tools-ides/intellij-plugin/using-intellij-plugin-features#checking-spellings)
-- [Analyzing semantics](tools-ides/intellij-plugin/using-intellij-plugin-features#analyzing-semantics)
-- [Code folding](tools-ides/intellij-plugin/using-intellij-plugin-features#code-folding)
+- [Running Ballerina programs](/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#running-ballerina-programs)
```

Comment:
we might need to remove the tool-ides part from the link and start the link with the version here it is v0-991

Reply:
We need not remove the "tools-ides" part in these links as we didn't remove the "tools-ides" folder.

Reply:
Fixed in the below pages:

/learn/tools-ides/intellij-plugin.md
v0-991/learn/tools-ides/intellij-plugin.md
v1-0/learn/tools-ides/intellij-plugin.md
v1-1/learn/tools-ides/intellij-plugin.md
/learn/tools-ides/vscode-plugin.md
v0-991/learn/tools-ides/vscode-plugin.md
v1-0/learn/tools-ides/vscode-plugin.md
v1-1/learn/tools-ides/vscode-plugin.md
/learn/tools-ides/intellij-plugin/using-the-intellij-plugin.md
v0-991/learn/tools-ides/intellij-plugin/using-the-intellij-plugin.md
v1-0/learn/tools-ides/intellij-plugin/using-the-intellij-plugin.md
v1-1/learn/tools-ides/intellij-plugin/using-the-intellij-plugin.md
v0-991/learn/tools-ides/intellij-plugin/using-intellij-plugin-features.md
v0-991/learn/tools-ides/intellij-plugin/testing-and-debugging-the-plugin.md
/learn/tools-ides/vscode-plugin/graphical-editor.md
v0-991/learn/tools-ides/vscode-plugin/graphical-editor.md
v1-0/learn/tools-ides/vscode-plugin/graphical-editor.md
v1-1/learn/tools-ides/vscode-plugin/graphical-editor.md
/learn/tools-ides/vscode-plugin/language-intelligence.md
v0-991/learn/tools-ides/vscode-plugin/language-intelligence.md
v1-0/learn/tools-ides/vscode-plugin/language-intelligence.md
v1-1/learn/tools-ides/vscode-plugin/language-intelligence.md
/learn/tools-ides/vscode-plugin/run-all-tests.md
v0-991/learn/tools-ides/vscode-plugin/run-all-tests.md
v1-0/learn/tools-ides/vscode-plugin/run-all-tests.md
v1-1/learn/tools-ides/vscode-plugin/run-all-tests.md
/learn/tools-ides/vscode-plugin/run-and-debug.md
v0-991/learn/tools-ides/vscode-plugin/run-and-debug.md
v1-0/learn/tools-ides/vscode-plugin/run-and-debug.md
v1-1/learn/tools-ides/vscode-plugin/run-and-debug.md